Uppsala/UppEt7.xml

```diff
@@ -0,0 +1,336 @@
+                                suggests to narrow this down to the 17th century.</desc>
+                                <date notBefore="1600" notAfter="1699">17th century.</date>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">Paper sticker on the outside of the front board. It is not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                    <q xml:lang="sv">Upsala Univ. Bibliotek<lb/>
+                                        Kat. no VII.<lb/>
+                                        Etiopisk handskrift n. 6<lb/>
+                                    skänkt 3/9 1894</q>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">Paper sticker on the inside of the front board. It is not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                    <q xml:lang="sv">Upsala Univ. Bibliotek<lb/>
+                                        Kat. no VII.<lb/>
+                                        O Etiop. 7</q>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <locus target="iir"/>
+                                    <desc type="Unclear">Note in Swedish, not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                    <q xml:lang="sv">Etiopisk handskrift n. 6<lb/>
+                                        skänkt d. 3 Sept. 1894 till Upsala Universitets<lb/>
+                                        Bibliotek af <persName ref="PRS2680Bergman">August Bergman</persName>, svenskt missionär
+                                        i <placeName ref="LOC1741Balaza">Belleza</placeName>, Asmara, i Abyssinien</q>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">Wooden boards.
+                                </decoNote>
```

Comment:
That is a good and clear description that follows the guidelines. You can add for clarity: "Two pairs of sewing stations".  Perhaps you could count the number of quires to "Collation" (I think they are easy to count).

Reply:
Thank you! I will add that.